services/Storage/storageAccounts/alerts.yaml

```diff
@@ -42,7 +42,7 @@
   description: The storage account will be throttled if throughput exceeds the account's
     tier limit. Increasing the file share or storage tier may be necessary.
   type: Metric
-  verified: false
+  verified: true
```

Comment:
Lets add this reference link to this alert, so that we have something. :)

https://learn.microsoft.com/en-us/azure/storage/files/files-monitoring-alerts#how-to-create-an-alert-if-a-file-share-is-throttled